pgpubsub/triggers.py

```diff
@@ -11,30 +11,27 @@ def get_func(self, model: Type[Model]):
         return f'''
             {self._build_payload(model)}
             {self._pre_notify()}
-            perform pg_notify('{self.name}', payload);
+            perform pg_notify('{self.name}', payload::text);
```

Comment:
I'm kind of confused why we convert back to a text here - can you elaborate?

Reply:
`payload` is jsonb now but `pg_notify` can only sent text so we need to convert to text.